<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="message_message_output">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>511</width>
      <height>211</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>消息接收窗口</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="message_input">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>消息输入</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_Btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_Btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>430</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP_textBrowser">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>151</width>
      <height>371</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>显示主机IP地址</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_Btn">
    <property name="geometry">
     <rect>
      <x>572</x>
      <y>30</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>启动服务器</string>
    </property>
   </widget>
   <widget class="QLabel" name="port_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>输入端口号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
